recipes/createrepo_c/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "createrepo_c" %}
+{% set version = "0.15.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rpm-software-management/createrepo_c/archive/{{ version }}/createrepo_c-{{ version }}.tar.gz
+  sha256: 41a6181885e38357f7ca843bf9dd27d398a02f79e3def8230e24fd643af11ce3
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - cmake
+    - pkg-config
+  host:
+    - python
+    - zchunk
+    - libcurl
+    - openssl
+    - xz
+    - libmagic
+    - sqlite
+    - libxml2
+    - glib
+    - bzip2
+    - rpm-tools
+    - drpm
+    - zlib
+    - libmodulemd
+    # Used for running tests
+    - nose
+  run:
+    - python
+    # https://abi-laboratory.pro/index.php?view=timeline&l=rpm
+    - {{ pin_compatible('rpm-tools', max_pin="x.x") }}
+    - {{ pin_compatible('libmodulemd', max_pin="x") }}
+
+test:
+  commands:
+    - createrepo_c --help
+
+about:
+  home: http://rpm-software-management.github.io/createrepo_c/
+  license: GPL-2.0-only
```

Comment:
You didn't ask for a review here yet, so feel free to ignore :)
Are you sure about this? It seems that they relicensed to GPL2+ in https://github.com/rpm-software-management/createrepo_c/commit/d9e6744cb2b82640bce045c357abdab965adbd4a.

Reply:
It's only a draft because of the other packages need to me merged first :) Yes it was, thank you for checking these thoroughly! 🍰